docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.6"
+
+services:
+  fcrepo:
+    image: fcrepo/fcrepo:latest
+    ports:
+      - "8080:8080"
+      - "8181:8181"
+      - "61616:61616"
+    environment:
+      CATALINA_OPTS: "-Dfcrepo.jms.baseUrl=http://fcrepo:8080"
+
+  solr:
+    image: solr:8
+    ports:
+      - "8983:8983"
+    command:
+      - solr-precreate
+      - fcrepo
+
+  fuseki:
+    image: atomgraph/fuseki
+    ports: 
+      - "3030:3030"
+    command:
+      - --mem 
+      - /fcrepo
+      
+  camel-toolbox:
+    build: .
+    image: fcrepo/fcrepo-camel-toolbox
```

Comment:
If we wanna build the image with docker-compose or not, depends on what we provide the docker-compose.yml for:

- for development/testing?
- as an example on how to integrate Fedora Repository, Camel Toolbox and surrounding services?

Reply:
The purpose of docker compose is two-fold: 
1) for demonstration in workshops and presentations
2) for manual testing (and possibly automated testing in the future.
3) a starter recipe for those wanting to use docker compose.

I don't use it for development, but that's just because  tend to run my debugger while developing on an instance running in local host.  Other services, such as fcrepo and solr I tend to launch as separate docker containers.

Reply:
> Then I would vote for not building the fcrepo/fcrepo-camel-toolbox image in the docker-compose.yml --> because normally you would want to pull the image from Docker Hub, right?

Agreed, my thought was once we have a fcrepo-camel-toolbox container published we can remove the build from docker-compose.yml (splitting up the Dockerfile and docker-compose PRs would probably have made more sense in this regard), though we could certainly do it before as well.